packages/components/src/internal/app/utils.ts

```diff
@@ -523,7 +535,12 @@ export function getMenuSectionConfigs(
         sectionConfigs = sectionConfigs.push(Map({ [REGISTRY_KEY]: getRegistrySectionConfig() }));
     }
     if (isBioOrSM) {
-        sectionConfigs = addSamplesSectionConfig(user, sectionConfigs);
+        const configs = Map<string, MenuSectionConfig>({ [SAMPLES_KEY]: getSamplesSectionConfig(user) }).asMutable();
```

Comment:
minor: remove usages of `asMutable` and `asImmutable`

Reply:
Removed.